esp_flasher/gui.py

```
@@ -352,8 +352,9 @@ def _reconnect_after_flash(self):
         """Reconnect to serial port after flash"""
         try:
             self.connect_to_port()
-        except Exception as e:
-            print(f"\033[31mFailed to reconnect: {e}\033[0m")
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)